.github/workflows/ci.yml

```diff
@@ -95,9 +95,17 @@ jobs:
         run: |
           pytest --cov-report=xml:coverage.xml --cov-fail-under=0
 
+          echo "Contents of coverage.xml after pytest:"
+          cat /home/runner/work/power-grid-model-io/power-grid-model-io/coverage.xml
+
           # Fix relative paths in coverage file
           # Known bug: https://community.sonarsource.com/t/sonar-on-github-actions-with-python-coverage-source-issue/36057
           sed -i 's@/home/runner/work/power-grid-model-io/power-grid-model-io@/github/workspace@g' coverage.xml
+      
+      - name: Verify updated coverage file
+        run: |
+          echo "Contents of coverage.xml after sed command:"
+          cat /github/workspace/coverage.xml
```

Comment:
seems like the working directory changed

https://github.com/PowerGridModel/power-grid-model-io/actions/runs/15045176657/job/42285816203?pr=302#step:5:142

Reply:
Where do you see it? What does it change to? 
That might explain this issue.

Reply:
no idea, all i saw was that it doesn't exist 😛